aggregator/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "aggregator"
+version = "0.1.0"
+authors = ["Cardinal Cryptography"]
+edition = "2021"
+license = "Apache 2.0"
+
+
+[dependencies]
+aleph-bft = "0.18.1"
```

Comment:
I'm pretty sure you are only using this for `Recipient`, so you probably want `aleph-bft-types` instead.

Reply:
done